src/cc/usdt/usdt.cc

```diff
@@ -31,7 +31,9 @@
 
 namespace USDT {
 
-Location::Location(uint64_t addr, const char *arg_fmt) : address_(addr) {
+Location::Location(uint64_t addr, std::string bin_path, const char *arg_fmt)
+    : address_(addr),
+      bin_path_(bin_path) {
```

Comment:
If we're not doing `std::move` from `bin_path`, it's wasteful to pass it by value -- there will be two copies for lvalue arguments.